index.bs

```diff
@@ -439,6 +440,157 @@ spec: webidl
     it MUST remove it.
     </li>
   </ol>
+
+  <section>
+    <h3 id="storage-manipulation">Storage Manipulation Algorithms</h3>
+
+    <p>
+      Other specifications can use the algorithms defined in this section to
+      manipulate the <a>permission store</a> from their algorithms.
+    </p>
+
+    <div class="example" id="example-storage-manipulation">
+      <p>
+        The [[notifications]] API can check for the user having granted
+        permission using:
+      </p>
+      <blockquote>
+        If the {{"notifications"}} <a>permission's state</a> is not
+        {{"granted"}}, reject <var>promise</var> with a {{TypeError}} exception
+        and terminate these substeps.
```

Comment:
The current realm is just ambiently available, I think. Nothing in WebIDL says it goes away when you call another algorithm.

The origin is the same for all three, so until folks start using other aspects of the settings object, entry/incumbent/relevant doesn't matter. We'll have to ask @raymeskhoury and @palmer if they've picked one for checking embedding.